examples/documentation/tests/bindings/test_documentation.rb

```diff
@@ -1,12 +1,10 @@
-require "test/unit"
-require "documentation"
- 
-class TestAdd < Test::Unit::TestCase
-  def test_hello
-    assert_equal(Documentation.hello(Documentation::Person.new("Tom")), "Hello Tom!")
-  end
+# frozen_string_literal: true
 
-  def test_add
-    assert_equal(5, Documentation.add(2, 3))
-  end
-end
+require 'test/unit'
+require 'documentation'
+
+include Test::Unit::Assertions
+
+assert_equal Documentation.hello(Documentation::Pet.new("Tom")), "Hello Tom!"
+assert_equal Documentation::Person.new("Daniel").get_name(), "Daniel"
+assert_equal Documentation.add(2, 4), 6
```

Comment:
Hmmm, that is pretty unusual approach to Ruby testing. Why you have removed while test class?

Reply:
They didn't work (weren't launched).
I don't know Ruby, so I don't know why, maybe `uniffi::build_foreign_language_testcases!` starts those files as scripts or something. 
I picked test structure from [here](https://github.com/eigerco/uniffi-rs/blob/main/examples/geometry/tests/bindings/test_geometry.rb).

If using class tests structure, maybe tests need to be started somehow?
Like in Python there is this section:
```py
if __name__ == '__main__':
    unittest.main()
```

Is there something like this for Ruby?

Reply:
Done.

Reply:
Leaving as is to match existing examples.